### PR TITLE
Add team field to people + add Asia Hub

### PR DESCRIPTION
**Pull request by willemarcel**
On Staff page:
- Use the country field only to store the information where the person lives
- Add the Team field to define if the person is member of a regional office or hub
- List Asia Hub
- Update information about some people

### CODE REVIEW

File: _people/hycinth-umaran.markdown [hunk 1]
```diff
@@ -5,8 +5,10 @@ Photo: "/uploads/mumaran-23a31f.jpg"
 Member Type:
   Is Staff: true
 Job Title: Scrum Master and Agile Coach
+Country: Philippines
+Team: Tech Team
```
Comment (dakotabenjamin) [inline _people/hycinth-umaran.markdown:9]: is the Tech Team supposed to be its own category on the staff page?

**Review by dakotabenjamin — APPROVED** [review]
Other than one question about Hya's team, everything works as intended.

File: _people/hycinth-umaran.markdown (hunk 1 above)
Comment (willemarcel) [reply _people/hycinth-umaran.markdown:9]: @dakotabenjamin my idea is to open the possibility to subdivide the Global staff by teams later, if desired, or show that information on the user page.

Comment (dakotabenjamin) [reply _people/hycinth-umaran.markdown:9]: I thought that might be the case. I like the idea.

**Pull request merged**